tests/integration/test_target_redshift.py

```diff
@@ -491,6 +491,30 @@ def test_custom_copy_options(self):
         target_redshift.persist_lines(self.config, tap_lines)
 
 
+    def test_copy_using_aws_environment(self):
+        """Test loading data with aws in the environment rather than explicitly provided access keys"""
+        tap_lines = test_utils.get_test_tap_lines('messages-with-three-streams.json')
+
+        try:
+            os.environ["AWS_ACCESS_KEY_ID"] = os.environ.get('TARGET_REDSHIFT_AWS_ACCESS_KEY')
+            os.environ["AWS_SECRET_ACCESS_KEY"] = os.environ.get('TARGET_REDSHIFT_AWS_SECRET_ACCESS_KEY')
+            config['aws_access_key_id'] = None
+            config['aws_secret_access_key'] = None
```

Comment:
this needs to be `self.config['aws_access_key_id']` and `self.config['aws_secret_access_key']`
Can you please change it?

Reply:
Done.